region/lowernorfair/east/Pillar Room.json

```diff
@@ -558,6 +558,39 @@
         "Acid damage is expected, but any mistakes greatly increases the time spent in acid."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode, Screw Attack",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "ScrewAttack",
+        {"or": [
+          "SpaceJump",
+          "canTrickyJump"
+        ]},
+        "h_HeatedGModeOpenDifferentDoor"
+      ],
+      "exitCondition": {
+        "leaveNormally": {}
+      },
+      "unlocksDoors": [
+        {
+          "types": ["missiles"],
+          "requires": [{"heatFrames": 50}, {"acidFrames": 50}]
+        },
+        {"types": ["super"], "requires": []},
+        {
+          "types": ["powerbomb"],
```

Comment:
Power bomb doors can be opened in Gmode, and you have the time from this direction.